lib/oci8/compat.rb

```diff
@@ -108,6 +108,9 @@ def self.select_number_as
 
   module BindType
     # alias to Integer for compatibility with ruby-oci8 1.0.
-    Fixnum = Integer
```

Comment:
Oops! I'll revert it the master code base.

https://github.com/kubo/ruby-oci8/blob/5c333cee82c807b591a40d3d2f6e7fd2cdd0db0f/lib/oci8/compat.rb#L113-L116